abci/types/types.pb.go

```diff
@@ -1235,11 +1235,11 @@ func (m *ResponseFlush) XXX_DiscardUnknown() {
 var xxx_messageInfo_ResponseFlush proto.InternalMessageInfo
 
 type ResponseInfo struct {
-	Data                 string   `protobuf:"bytes,1,opt,name=data,proto3" json:"data,omitempty"`
```

Comment:
This file is auto generated by protoc, so your changes will be overwritten next time.

Solution we came up with a while ago is to use custom marshallers - https://github.com/tendermint/tendermint/issues/3882. Is this not working?

Reply:
Yes, you are correct @melekes. I have added `omitempty` again.